lib/tentacat/issues/comments.ex

```diff
@@ -3,7 +3,9 @@ defmodule Tentacat.Issues.Comments do
   alias Tentacat.Client
 
   @doc """
-  List comments on an issue. Issue Comments are ordered by ascending ID.
+  List comments on an issue.
+
+  Issue Comments are ordered by ascending ID.
```

Comment:
hmm, not sure if we need to split like this for all one liners. Doesn't it add to many white spaces to doc?